CATALOG.md

```diff
@@ -325,6 +325,19 @@ Exception Process|There is no documented exception process for this.
 
 ### lifecycle
 
+#### colocation-allowed
+
+Property|Description
+---|---
+Test Case Name|colocation-allowed
+Test Case Label|lifecycle-colocation-allowed
```

Comment:
test name is now `affinity-required`, I believe

Reply:
Thanks, just regenerated the CATALOG.md.